src/shipit_pulse_listener/shipit_pulse_listener/listener.py

```diff
@@ -34,11 +34,14 @@ def __init__(self, configuration):
         # Start on first page
         self.before = None
 
+        # Limit to diffs on last 12 hours
+        self.date_limit = datetime.now() - timedelta(hours=12)
+
     def list_differential(self):
         '''
         List new differential items
         '''
-        url = '{}/differential.revision.search'.format(self.api_url)
+        url = '{}/differential.diff.search'.format(self.api_url)
```

Comment:
We can probably use some query parameter to only return diffs from past 12 hours instead of getting them all and filtering out locally

Reply:
Does not look like it does: https://phabricator.services.mozilla.com/conduit/method/differential.diff.search/